README.md

```diff
@@ -1,24 +1,139 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [📹 video presentation](#video-presentation)
+  - [:camera: screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* Ruby version
+- [📝 License](#license)
 
-* System dependencies
+# 💰 Greetings App Back-end <a name="about-project"></a>
 
-* Configuration
+> This is a starting point to create a React front-end with a Rails back-end and connect them to display a random message
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
 
-* How to run the test suite
+## Link to the Front-end repository
 
-* Services (job queues, cache servers, search engines, etc.)
+[Hello_rails_front-end](https://github.com/Chaseknis/helllo_rails_front-end.git)
 
```

Comment:
- [ ] Same here, the Link to the Front-end repository is broken. 👇🏻 Please fix this. 💯 👍🏻 

![image](https://github.com/Chaseknis/helllo_rails_back-end/assets/79058364/5e0affa2-7e7c-4494-a0b6-f566a77e4317)

Reply:
fixed